cypress/integration/api-docs-smoke-test/api-type-soft-linking.js

```diff
@@ -0,0 +1,14 @@
+import { URL_API_DOCS_SMOKE_TEST } from '../../support/urls';
+
+describe('API type soft linking', () => {
+  it('should go to page of a type, then find the page of linked property type', () => {
+    cy.visit(URL_API_DOCS_SMOKE_TEST);
+    cy.findByLabelText('Open ObjectTestType').should('exist');
```

Comment:
Using `findBy` implicitly checks for the element existence, so there is no need to specify that. You can just click on it.